test/Microsoft.Framework.Configuration.Binder.Test/ConfigurationBinderTests.cs

```diff
@@ -178,11 +221,17 @@ public void CanReadComplexProperties()
             };
             var builder = new ConfigurationBuilder(new MemoryConfigurationSource(dic));
             var config = builder.Build();
-            var options = new ComplexOptions();
-            config.Bind(options);
+
```

Comment:
Nit: its worth considering splitting these two tests BindCanReadComplexProperties / GetCanReadComplexProperties.  Slightly cleaner for this and the one below